openshift/f8a-notification.yaml

```diff
@@ -14,6 +14,8 @@ objects:
     annotations:
       description: f8a-notification-scheduler
   spec:
+    successfulJobsHistoryLimit: 1
+    failedJobsHistoryLimit: 1
```

Comment:
Nice :wink: Did it help?

Reply:
Yep :D, but we found a bug in Openshift, looks like even though `failedJobsHistoryLimit` is set to `1`. It never deletes the older failed pods.
![screenshot from 2018-08-10 11-50-29](https://user-images.githubusercontent.com/8664238/43948011-385d35f2-9ca7-11e8-81ba-ac478d7bcb1a.png)